

==> WARNING: A newer version of conda exists. <==
  current version: 4.12.0
  latest version: 23.7.2

Please update conda by running

    $ conda update -n base -c defaults conda


 /home/yanw/.conda/envs/uncertain/lib/python3.8/site-packages/paramz/transformations.py:111: RuntimeWarning:overflow encountered in expm1
 /home/yanw/.conda/envs/uncertain/lib/python3.8/site-packages/GPy/kern/src/rbf.py:52: RuntimeWarning:overflow encountered in square
Traceback (most recent call last):
  File "main.py", line 153, in <module>
    print('MSPE prediction prior:', (np.sum(prior_prediction_variance)+np.square(np.linalg.norm(y-np.mean(y_final_mean_list_prior,axis=0))))/100)
ValueError: operands could not be broadcast together with shapes (100,) (104,) 
--------------------------------------------------------------------------
Primary job  terminated normally, but 1 process returned
a non-zero exit code. Per user-direction, the job has been aborted.
--------------------------------------------------------------------------
--------------------------------------------------------------------------
mpirun detected that one or more processes exited with non-zero status, thus causing
the job to be terminated. The first process to do so was:

  Process name: [[20237,1],0]
  Exit code:    1
--------------------------------------------------------------------------
cp: missing destination file operand after '/home/yanw/uncertain'
Try 'cp --help' for more information.
